app/scripts/modules/core/application/config/applicationConfig.view.html

```diff
@@ -15,17 +15,17 @@
     <page-section key="cache" label="Cache Management">
       <application-cache-management application="config.application"></application-cache-management>
     </page-section>
-    <page-section key="chaos" label="Chaos Monkey" ng-if="config.feature.chaosMonkey">
-      <chaos-monkey-config application="config.application"></chaos-monkey-config>
-    </page-section>
     <page-section key="traffic-guards" label="Traffic Guards">
       <traffic-guard-config application="config.application"></traffic-guard-config>
     </page-section>
-    <page-section key="snapshot" label="Serialize Application" ng-if="config.feature.snapshots">
+    <page-section key="snapshot" label="Serialize Application" visible="config.feature.snapshots">
       <application-snapshot-section application="config.application"></application-snapshot-section>
     </page-section>
     <page-section key="delete" label="Delete Application">
       <delete-application-section application="config.application"></delete-application-section>
     </page-section>
+    <page-section key="chaos" label="Chaos Monkey" visible="config.feature.chaosMonkey">
```

Comment:
Is this section going to go under the `Delete Application` section? I would try to match the ordering of https://github.com/spinnaker/deck/blob/master/app/scripts/modules/netflix/application/applicationConfig.html#L21-L29 as much as possible.